examples/text_to_image/requirements.txt

```diff
@@ -0,0 +1,6 @@
+accelerate
```

Comment:
going forward I think we should also pin `diffusers` here or make sure required version is always updated